com/course/zenkouv/src/Lecture3/Task1/factory/outfitFactory/Blacksmith.java

```diff
@@ -0,0 +1,58 @@
+package Lecture3.Task1.factory.outfitFactory;
+
+import Lecture3.Task1.model.Material;
+import Lecture3.Task1.model.Outfit;
+import Lecture3.Task1.model.OutfitType;
+
+public class Blacksmith {
+
+    private OutfitFactory factory;
```

Comment:
better - add two fields 
private static final OutfitFactory ArmorFactory =  new ArmorFactory()
private static final  OutfitFactory WeaponFactory = new WeaponFactory()
and return in switch related prepared factory